dev/fattest.simplicity/src/componenttest/rules/repeater/MicroProfileActions.java

```diff
@@ -382,8 +382,8 @@ public class MicroProfileActions {
     public static final FeatureSet MP60 = new FeatureSet(MP60_ID, MP60_FEATURE_SET, EEVersion.EE10);
     public static final FeatureSet MP61 = new FeatureSet(MP61_ID, MP61_FEATURE_SET, EEVersion.EE10);
     public static final FeatureSet MP70_EE10 = new FeatureSet(MP70_EE10_ID, MP70_EE10_FEATURE_SET, EEVersion.EE10);
-    public static final FeatureSet MP70_EE11 = new FeatureSet(MP70_EE11_ID, MP70_EE11_FEATURE_SET, EEVersion.EE11);
-    public static final FeatureSet MP70_EE11_APP_MODE = new FeatureSet(MP70_EE11_APP_MODE_ID, MP70_EE11_FEATURE_SET, EEVersion.EE11);
+    public static final FeatureSet MP70_EE11 = new FeatureSet(MP70_EE11_ID, MP70_EE11_FEATURE_SET, EEVersion.EE11, EEVersion.EE11.getMinJavaLevel(), true);
+    public static final FeatureSet MP70_EE11_APP_MODE = new FeatureSet(MP70_EE11_APP_MODE_ID, MP70_EE11_FEATURE_SET, EEVersion.EE11, EEVersion.EE11.getMinJavaLevel(), true);
```

Comment:
I'm not sure that this is the right place to set this.

I think you could justify setting it only for the tests that it, or saying that all tests should check it.

Reply:
In general, I don't like adding this check at all. There's too great a risk that tests will not run when they should.

IMO it should be opt-in and only for the tests that really need it.

In some ways it would be better to have it as a check in the test rather than part of the repeat action at all.

Reply:
I considered that, I think I could put a second property in SkipForRepeat like `onlyIfFeatureNotPresent`. I'm not a fan of that myself, the SkipForRepeat code is brittle and has caused us problems before with the way it uses string suffixes as pusdo subclasses; I'd prefer to avoid it. 

But I'll draft you up a prototype to compare